src/platform/packages/shared/response-ops/rule_form/src/rule_details/rule_details.tsx

```diff
@@ -113,7 +136,15 @@ export const RuleDetails = () => {
           </EuiFormRow>
         </EuiFlexItem>
       </EuiFlexGroup>
+      <EuiSpacer size="l" />
+      <EuiFormRow fullWidth label={RULE_INVESTIGATION_GUIDE_LABEL} labelAppend={OptionalFieldLabel}>
```

Comment:
Is there a simple way to add the (?) icon with the hover tooltip that appears in other parts of these rule forms, so we can add a bit of context re: how this works? I was thinking something like 

"These details will be included in a new tab on the alert details page for every alert triggered by this rule"

Reply:
Although maybe it's easier to just use some kind of "description text" that appears as the lighter text under the input ... whatever works is fine but it would be nice to help users understand where this information will appear, I think?

Reply:
It would be nice to do the same for the Related Dashboards input below too, so maybe this can be a follow up to handle it for both ...